app/src/main/java/org/cru/godtools/fragment/ToolDetailsFragment.java

```diff
@@ -308,6 +331,24 @@ private void updateViews() {
         }
     }
 
+    private void updateOverviewVideo() {
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.JELLY_BEAN_MR1) {
+            if (mVideoBanner != null) {
```

Comment:
Is this guaranteed from the above call? Or perhaps its not because its Android and is a class attribute?

Reply:
@wrandall22 I placed this null check for reusability